File: lesson_22_java/src/main/java/com/lesson22java/controllers/FrontAboutController.java
package com.lesson22java.controllers;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping("/front")
public class FrontAboutController {
    @GetMapping("/about")
    public String about() {
        return "about-page";
    }
}

File: lesson_22/src/main/kotlin/com/lesson22/controllers/FrontAboutController.kt
package com.lesson22.controllers

import com.lesson22.services.ProductService
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@RequestMapping("/front")
class FrontAboutController (val productService: ProductService) {
    @GetMapping("/about")
    fun about() : String {
        return "about-page"
    }
}